draft-wussler-openpgp-forwarding.md

```diff
@@ -80,6 +84,9 @@ In this document we outline the changes necessary to the OpenPGP protocol to saf
   - Forwardees to read the transformed email.
 
 This is achieved using a BBS-like transformation {{EUROCRYPT}} on the ECDH
+// Is there a less cryptic name for this? This might not mean much to people,
+// and also for me the primary association of "BBS" is "BBS signatures", though
+// it seems these are a different B, B and S :)
 encryption algorithm using Curve25519.
```

Comment:
How about something like
```suggestion
This is achieved by transforming messages encrypted using Curve25519,
with a scheme similar to the one described in {{EUROCRYPT}}.
```
Or, alternatively, since you wrote a paper about it, I think you can also quote yourself here instead, and work that title into the sentence, e.g.

```suggestion
This is achieved by diverting the ECDH key exchange of messages
encrypted using Curve25519, as described in {{FORWARDING}}.
```